packages/datepicker/src/react/calendar.tsx

```diff
@@ -103,6 +119,16 @@ export const Calendar = React.forwardRef<HTMLDivElement, CalendarProps>(
         return () => el.removeEventListener('animationend', updateOffset)
       }
     }, [])
+    const gridLabels = calendars.map((calendar, i) =>
```

Comment:
`monthIds` seems more clear.

Reply:
I think code readability is usually a top priority. I was making a suggestion toward readability.

My thought flowed from this fact:

From the [docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute):

> The aria-labelledby attribute establishes relationships between objects and their label(s), and its value should be one or more element IDs.

Seem like they're ids, and they're applied to the element that is labelling a month. I didn't know what grid was.